core/jobs.py

```diff
@@ -127,11 +127,13 @@ def _create_new_job():
         return transaction_services.run_in_transaction(_create_new_job)
 
     @classmethod
-    def enqueue(cls, job_id, additional_job_params=None):
+    def enqueue(cls, job_id, queue_name='default', additional_job_params=None):
```

Comment:
Here and elsewhere, avoid magic strings, use named constants in taskqueue_services.

Also, what do you think about not allowing a default arg here, and forcing clients to specify the queue explicitly? I think it's quite client-dependent really so there's probably no need for a default. I also suggest that we declare a new queue for 'continuous_computations' that's set to the low rate.